src/Settings/RMCParameters.as

```diff
@@ -108,6 +112,11 @@ namespace PluginSettings
             RMC_PrepatchTagsWarns = UI::Checkbox("Prepatch maps warnings", RMC_PrepatchTagsWarns);
             UI::SetPreviousTooltip("Display a warning if the map is built before the new physics patches (eg the bobsleigh update)");
 #endif
+
+            UI::SetNextItemWidth(300);
+            RMC_MX_Url = UI::InputText("ManiaExchange Proxy Url", RMC_MX_Url);
+            UI::SetPreviousTooltip("Proxy api calls to ManiaExchange via this url. Useful for caching and preloading api responses for better performance.");
```

Comment:
Change "Proxy URL" by "Base URL" (the same way as Openplanet's Plugin Manager did)

Reply:
updated the text to remove the word proxy and just use "base"